### ISSUE TITLE
You add +60 secs on every /bonus call

### ISSUE THREAD

**Issue by bots-business** [Untoldhacker-Dev/friendly-octo-fortnight#65]
Line: 

`cooldown.set(60 + cooldown.value()) // add 60 sec to cooldown`

why do you need this?
this code add +60 secs for waiting

**Comment by bots-business** [comment]
```
// get current cooldown res for chat
let cooldown = Libs.CooldownLib.chat.getCooldown("BonusCooldown")

// for user:
// let cooldown = Libs.CooldownLib.user.getCooldown("GemBonusCooldown");

cooldown.value() // current cooldown in second
```

I think this must be removed

**Comment by bots-business** [comment]
Please do not close issue after fixing. Assign tag

**Comment by Untoldhacker-Dev** [comment]
Fixed: #66
